### ISSUE TITLE
Xdebug can not be automatically enabled

### ISSUE THREAD

**Issue by Pen-y-Fan** [leokhoa/laragon#332]
When Xdebug is downloaded* and copied to the ext directory, when switching to Lagaon and try to enable in **Extensions** or **Quick settings**,  Laragon does not automatically add the Xdebug extension. 

It is listed in the list of **Extentions**, when clicked the menu closes. If Apache is running it is restarted. However, the extension isn't added to php.ini

The workaround is to manually add the extension to **php.ini** :

```ini
[xdebug]
zend_extension=xdebug-3.1.5-8.1-vs16-x86_64
xdebug.mode = coverage,debug,develop
```

Laragon will then toggle the Xdebug extension on and off via **Extensions** or **Quick settings** (add and remove a **;**)

This was also a problem with Laragon 5, it was brought up on the forum, but I guess no one raised it as a GitHub issue 🤷‍♂️

- [\[Tutorial\] How to add Xdebug to Laragon](https://forum.laragon.org/topic/264/tutorial-how-to-add-xdebug-to-laragon/28)

I'm not 100% sure when it broke, possibly around  3.1.0,  there is a post stating it works with Xdebug 3.0.4.

* the Windows dll for 3.1.5 is under [historical downloads](https://xdebug.org/download/historical), it looks like the alpha version doesn't have a Windows build.

**Comment by rforced** [comment]
Are you able to enable any of the other php extensions? On Laragon 6 I cannot enable any.

**Comment by Pen-y-Fan** [comment]
> Are you able to enable any of the other PHP extensions? On Laragon 6 I cannot enable any.

I tried **pdo_sqlite** and it enabled straight away.  🎉

However I also tried **dl_test** and **zend_test**, not sure what there extensions are, nothing enabled 🤷‍♂️ (see screenshot)

[Image: PHP extensions](https://i.imgur.com/Ee2w6Ss.png)

**Comment by rforced** [comment]
Yea, I tried Redis extension as well, doesn't enable at all.

**Comment by msiemaszko** [comment]
To proper enable/disable extension by laragon you should defined extensions in php.ini without "php_" prefix, even if your manually downloaded library have prefix in name 

ex. for php_xdebug-3.1.5-8.1-vs16-nts-x86_64.dll define in php.ini 
```
[xdebug]
zend_extension=xdebug-3.1.5-8.1-vs16-nts-x86_64 
```

**Comment by rforced** [comment]
@msiemaszko yes, you can do it manually. However one of Laragon's features is to enable and disable extensions from the right click menu. That's what this issue is about.

**Comment by juanmalunatic** [comment]
@rforced 

I tried @msiemaszko's comment and it works fine. Did you give the instructions a try? 

When you enable the extension manually you don't remove the "php" prefix. If you do this, Laragon's right click menu to enable/disable won't work.

When you do as @msiemaszko suggested, removing the prefix, the extension **will** be managed by Laragon's menu.

**Comment by jakubtro13** [comment]
Hi,
The best methods was on official web page : https://xdebug.org/docs/install#windows
Result of installation wizard example on my Win10 and my php.ini :
Download [php_xdebug-3.2.0-8.1-vs16-nts-x86_64.dll](https://xdebug.org/files/php_xdebug-3.2.0-8.1-vs16-nts-x86_64.dll)
    Move the downloaded file to F:\laragon\bin\php\php-8.1.12-nts-Win32-vs16-x64\ext, and rename it to php_xdebug.dll
    Update F:\laragon\bin\php\php-8.1.12-nts-Win32-vs16-x64\php.ini and add the line:
    zend_extension = xdebug

**Comment by Pen-y-Fan** [comment]
@jakubtro13 the last step is incomplete, the php.ini needs a minimum of two lines to work, or with Nginx, it  needs three:

```ini
[xdebug]
; these two lines are required for xDebug 3+
zend_extension = xdebug
xdebug.mode=coverage,debug,develop
; Laragon has a conflict with Nginx and xDebug using port 9003, set xDebug to use 9000.
xdebug.client_port=9000
```

Prior to xDebug 3.0.4 all we needed to do was steps one and two: download and copy to the ext directory., Laragon used to automatically detect and add the `zend_extension = xdebug.....` to php.ini.

I tried removing my xDebug setting from the **php.ini** and renaming the DLL to **php_xdebug.dll**. Laragon will show the xDebug option in **PHP > quick settings > xdebug** but will not automatically add it to the **php.ini** as it used to do.

Once the above lines are added to the php.ini, then Laragon will happily toggle xDebug on and off.

I'm not even sure if @leokhoa can add/fix the feature. Previously Laragon would only add the extension to the ini file. From xDebug 3 the second line is also required to enable the required mode.
